drivers/adc/Kconfig.stm32f4

```diff
@@ -0,0 +1,373 @@
+# Kconfig - ADC configuration options
+
+#
+# Copyright (c) 2015 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ADC_STM32F4
+	bool "STM32F4 ADC driver"
+	depends on SOC_FAMILY_STM32
+	select USE_STM32_HAL_GPIO
+	select USE_STM32_HAL_GPIO_EX
+	select USE_STM32_HAL_ADC
+	select USE_STM32_HAL_ADC_EX
+	select USE_STM32_HAL_CORTEX
+	help
+		Enable the driver implementation for the stm32f4xx ADC
```

Comment:
Text of 'help' section should be indented by 2 spaces (you can look at other files as exemple)